firecloud/fiss.py

```diff
@@ -859,23 +859,42 @@ def attr_get(args):
     a special __header__ entry is optionally added to the result. '''
 
     if args.entity_type and args.entity:
-        r = fapi.get_entity(args.project, args.workspace, args.entity_type, args.entity)
+        if args.entity_type == "ref":       # return referenceData (b37 or hg38) attributes
```

Comment:
Currently in the Reference Data section of the Data Tab, there are only 2 options a user can select (hg38 or b37Human). When a user selects either of these options, the UI sends a request to the Terra backend to save the the reference data as entities and appends `referenceData_`  to the entity name and it is returned with this prefix by `fissfc attr_list`. The only way to add additional options to that drop-down menu is to this [file](https://github.com/DataBiosphere/terra-ui/blob/dev/src/data/reference-data.js). Otherwise, if a user wanted to add reference files outside of those provided, they would do so from the Workspace Data section - anything listed here is returned by `fissfc attr_list` without the `referenceData_` prefix; it is returned by whatever key is given in Workspace Data. I confirm that the UI disallows a user to manually add the `referenceData_` prefix from Workspace Data (though it does allow from the API). 

Perhaps I need to find a clearer way of describing that "ref" items are those that appear within the Reference Data section only? I'm happy to discuss further if I've misunderstood anything!

Reply:
A user can manually add a reference to this section via `[fissfc attr_set](https://github.com/broadinstitute/fiss/blob/a3a51b7cfd72d7527810b8569d36ba923b7d0b72/firecloud/fiss.py#L920)` and creating workspace attributes with names following the pattern `referenceData_<ref_name>_<attr_name>` - in [this workspace](https://app.terra.bio/#workspaces/broad-firecloud-gdac/GDC_WGS_pipeline_test/data), I did this to create the GRCh38 reference.

My reasoning is that if someone is enough of a power user to be using `FISS`, then they are enough of a power user to add a custom reference, and would want any tool that made use of it to reflect that in its flexibility.